clonelists/hash.json

```diff
@@ -53,7 +53,7 @@
 	"Nintendo - Nintendo 3DS (No-Intro).json": "486d484cd3e464352941ea007035bde455e82c87269302099c9c12ee00c201ea",
 	"Nintendo - Nintendo 64 (No-Intro).json": "fe697b1a52506520d2cecaab2bd9ef1745cc90e8462024697988f33e3bc5c2a3",
 	"Nintendo - Nintendo 64DD (No-Intro).json": "5318a02a96be9e192a56732178c7212e4fff308b02fd4efa4750f40f93d52401",
-	"Nintendo - Nintendo DS (No-Intro).json": "1a62b75bf772e1666a9011c22806c3108a5437b420ea35420754ee1c8bee7856",
+	"Nintendo - Nintendo DS (No-Intro).json": "1a5ed51cdac75fc484bab2f3a9b31465e2753365afc42a0de2097ae54daee813",
```

Comment:
I'm getting a different hash here (`0838d86aee8f6b64abd2f102119b0eba0b3b59d27b8d8e77126df29e6057be01`). The only thing I can think of that might be messing things up are line endings -- they're meant to be purely LF (although I can't tell from this PR alone what line endings are in use).

It looks like I can force LF line endings on the repo itself, but that's not going to help calculating the hash in this file. For future submissions I'll update `hash.json` separately on my side until I can put tools and processes in people's hands to ensure consistency.